commands/start.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+// Do this as the first thing so that any code reading it knows the right env.
+process.env.BABEL_ENV = 'development';
+process.env.NODE_ENV = 'development';
```

Comment:
These can be adjusted by the CLI arguments, right?

Reply:
I guess this won't work if someone adjusts with CLI args, I have fixed this behaviour in the latest commit.